server/channels/web/oauth_test.go

```diff
@@ -417,32 +417,36 @@ func TestMobileLoginWithOAuth(t *testing.T) {
 		responseWriter := httptest.NewRecorder()
 		request, _ := http.NewRequest(http.MethodGet, th.App.GetSiteURL()+"/oauth/gitlab/mobile_login?redirect_to="+url.QueryEscape("mmauth://"), nil)
 		mobileLoginWithOAuth(c, responseWriter, request)
-		assert.Contains(t, responseWriter.Body.String(), "mmauth://")
+		assert.Equal(t, responseWriter.Code, 302)
```

Comment:
The test case description suggests the redirect URL should still be present in the output -- should we update the description, or add back the `mmauth://` check?

Reply:
Will update the description as the method will redirect to the sso login page when the scheme is valid